modules/channel-web/src/views/lite/store/index.ts

```diff
@@ -407,6 +417,19 @@ class RootStore {
     })
   }
 
+  @action.bound
+  private emptyDelayedMessagesQueue(removeAll: boolean) {
+    while (this.delayedMessages.length) {
+      const message = this.delayedMessages[0]
```

Comment:
why not shift here instead as the return of shift is the first value of the array thus delayedMessages[0]

Reply:
Shift would remove the element from the array which should only be done if the if statement passes